frontend/src/components/ArtifactList/index.js

```
@@ -0,0 +1,128 @@
+import {Component} from 'react'
+import {connect} from "react-redux";
+import {componentsOperations} from "../duck";
+import artifactList from "./presenter.jsx"
+import get from "lodash/get"
+import queryString from "query-string";
+import history from "../../utils/history";
+import {isPrintableArtifact} from "../common";
+
+const mapStateToProps = (state) => {
```

ESLint

[error] Block must not be padded by blank lines.

(padded-blocks)